js/components/ImportIndicatorsPopover.js

```diff
@@ -525,10 +534,22 @@ export const ImportIndicatorsPopover = ({ page, program_id, tierLevelsUsed, stor
                         return (
                             <div className="import-confirm">
                                 <div className="import-confirm-text">
-                                    {displayError.view !== CONFIRM ?
+                                    {displayError.view === CONFIRM ?
+                                        <div>
+                                            {displayError.view === CONFIRM && displayError.error.indexOf(5) === -1 &&
```

Comment:
Is `displayError.view === CONFIRM` necessary here since it's wrapped in the same statement above?

What is the `indexOf(5)` checking for (maybe include a comment in the code on that).  I thought it might be looking for errors that aren't 500 errors, but it looks like `displayError.error` is HTML?

Reply:
I commented this line out. It's for the multiple uploader scenario, indexOf(5) refers to the error message id 5.